openedx_proversity_reports/edxapp_wrapper/get_modulestore.py

```diff
@@ -10,3 +10,11 @@ def get_modulestore(*args, **kwargs):
     backend = import_module(backend_function)
 
     return backend.get_modulestore_backend(*args, **kwargs)
+
+def item_not_found_error():
```

Comment:
I think it is better, since it is an exception, I mean, the exception could be raised or catched, and the error get_item not_found sounds as if we were assigning it to something... @andrey-canon 
Does it make sense? @andrey-canon